apps/financial-aid/README.md

```diff
@@ -31,6 +31,22 @@ Run Ósk or Veita client:
 
 Go to localhost:4200
 
+## To test authentication locally:
+
+Install https://github.com/cameronhunter/local-ssl-proxy:
+
+- `npm install -g local-ssl-proxy`
+
+- change defaultcookie in financial-aid/api/src/app/modules/auth/auth.controller.ts:
+
+  const defaultCookieOptions: CookieOptions = {
+  secure: true,
+  }
+
+- add .env to web project and change port
```

Comment:
Should we say the port explicitly since on the next line we are expecting the user to have osk running on port 4202 and the proxy on 4200?